README.md

```diff
@@ -40,25 +100,79 @@ Run `compileAndBuildVLCKit.sh` with the `-a ${ARCH}` option
 
     `cd vlc`
 
-    `git am ../../Resources/MobileVLCKit/patches/* `
+    `git am ../../Resources/MobileVLCKit/patches/*`
 
 3. run `compileAndBuildVLCKit.sh` with the `-n` and the `-a ${ARCH}` option
 
-## Get in touch!
+## Contribute
+
+As VLCKit is an open-source project hosted by VideoLAN, we happily welcome all kinds of contributions.
+
+### Pull Request
+
+Pull requests are more than welcome! If you do submit one, please make sure to use a descriptive title and description.
+
+### GitLab Issues
+
+You can look through the currently open issues on [GitLab] and choose the one that interests you the most.
+
+### Patches
+
+If you like the more classic apporach, you can submit patches!
+
+For detailed explanation on how to do so, please read our wiki page on [how to send patches].
 
-We happily provide guidance on VLCKit. The [web forum] is always there for you.
 
-If you prefer live interaction, reach out to us via our IRC channel on the [freenode] Network (irc.freenode.org, #videolan). Use the [Freenode Web] interface, if you don't have an IRC client at hand.
+## FAQ
+
+> Q. VLCKit being an open-source software, what does this mean for me and the end-user? And wasn’t MobileVLC removed from the App Store in 2011 for some crazy licensing reason?
+
+VLCKit being open-source means that you get access to the whole stack. There is no blackbox. All the source code is there at your fingertips. No reverse-engineering is needed and there are no private APIs.
+
```

Comment:
Since we don't explain why we got removed from the App Store can we remove this bit up here ? Also how open source works doesn't need to be in a README :)  @fkuehne ? okay with that ?